rent_golem/backend/requirements.txt

```diff
@@ -7,3 +7,5 @@ django-celery-beat==2.5.0
 redis==4.6.0
 yapapi==0.11.0
 django-extensions==3.2.3
+djangorestframework-simplejwt==5.3.0
+djangorestframework-simplejwt[crypto]
```

Comment:
I think `djangorestframework-simplejwt[crypto]` is optional and not needed in our case. Please check if tests work without it.